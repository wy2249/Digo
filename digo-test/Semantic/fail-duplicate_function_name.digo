func digo_main() void {
  
}

func foo() void {
  return 1
}

func foo() void{
  return 0
}