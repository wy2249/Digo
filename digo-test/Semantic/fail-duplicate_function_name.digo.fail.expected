Fatal error: exception Failure("Semant Err: duplicate function foo")
